packages/manager/core/generator/app/templates/src/components/GuideLink/guide-utils.tsx

```diff
@@ -0,0 +1,121 @@
+import { useState, useEffect } from 'react';
+import { Region } from '@ovh-ux/manager-config';
+import {
+  useAuthentication,
+  useEnvironment,
+} from '@ovh-ux/manager-react-core-application';
+
+const docUrl = 'https://docs.ovh.com/';
+
+type BaseUrlProps = {
+  [key in Region]: any;
+};
+
+const baseUrlPrefix: BaseUrlProps = {
+  EU: {
+    DEFAULT: `${docUrl}gb/en/`,
+    DE: `${docUrl}de`,
+    ES: `${docUrl}es`,
+    IE: `${docUrl}ie/en`,
+    IT: `${docUrl}it`,
+    PL: `${docUrl}pl`,
+    PT: `${docUrl}pt`,
+    FR: `${docUrl}fr`,
+    GB: `${docUrl}gb/en`,
+  },
+  CA: {
+    DEFAULT: `${docUrl}gb/en`,
+    CA: `${docUrl}ca/en`,
+    QC: `${docUrl}ca/fr`,
+    WE: `${docUrl}us/en`,
+    WS: `${docUrl}us/en`,
+  },
+  US: {
+    DEFAULT: `${docUrl}gb/en`,
+    US: `${docUrl}us/en`,
+  },
+};
+
+const GuidesList: any = {
+  guideLink1: {
+    DEFAULT: '/update-path',
+    DE: '/update-path',
+    ES: '/update-path',
+    IE: '/en/update-path',
+    IT: '/update-path',
+    PL: '/update-path',
+    PT: '/update-path',
+    FR: '/update-path',
+    GB: '/update-path',
+    CA: '/update-path',
+    QC: '/update-path',
+    WE: '/update-path',
+    WS: '/update-path',
+    US: '/update-path',
+  },
+  guideLink2: {
+    DEFAULT: '/guide-link-2-path',
+    DE: '/guide-link-2-path',
+    ES: '/guide-link-2-path',
+    IE: '/en/guide-link-2-path',
+    IT: '/guide-link-2-path',
+    PL: '/guide-link-2-path',
+    PT: '/guide-link-2-path',
+    FR: '/guide-link-2-path',
+    GB: '/guide-link-2-path',
+    CA: '/update-path',
+    QC: '/update-path',
+    WE: '/update-path',
+    WS: '/update-path',
+    US: '/update-path',
+  },
+  guideLink3: {
+    DEFAULT: '/guide-link-3-path',
+    DE: '/guide-link-3-path',
+    ES: '/guide-link-3-path',
+    IE: '/en/guide-link-3-path',
+    IT: '/guide-link-3-path',
+    PL: '/guide-link-3-path',
+    PT: '/guide-link-3-path',
+    FR: '/guide-link-3-path',
+    GB: '/guide-link-3-path',
+    CA: '/update-path',
+    QC: '/update-path',
+    WE: '/update-path',
+    WS: '/update-path',
+    US: '/update-path',
+  },
+};
+
+type getGuideLinkProps = {
+  name: string;
+  region: Region;
+  subsidiary: string;
+};
+
+function getGuideLink({ name, region, subsidiary }: getGuideLinkProps) {
+  try {
+    return (
+      `${baseUrlPrefix[region][subsidiary]}${GuidesList[name][subsidiary]}` ||
+      `${baseUrlPrefix[region][subsidiary]}${GuidesList[name]['DEFAULT']}`
+    );
+  } catch {
+    return `undefined`;
+  }
+}
+
+function useGuideUtils({ name }: any) {
+  const environment = useEnvironment();
+  const region = environment.getRegion();
+  const { subsidiary } = useAuthentication();
+  const [link, setLink] = useState(name);
+
+  useEffect(() => {
+    const tmpLink = getGuideLink({ name, region, subsidiary });
+    setLink(tmpLink);
+  }, []);
```

Comment:
```suggestion
  }, [name, region, subsidiary]);
```

To update the links if the language change, if someday we can change language without reloading the whole app